GlobalBehaviorandComponents/manageapps.py

```diff
@@ -87,12 +94,14 @@ def gbac_apps_delete():
 def gbac_apps_update():
     logger.debug("gbac_apps_update()")
     # user_info = get_userinfo()
-    # okta_admin = OktaAdmin(session[SESSION_INSTANCE_SETTINGS_KEY])
+    okta_admin = OktaAdmin(session[SESSION_INSTANCE_SETTINGS_KEY])
 
-    # app_name = request.args.get('appname')
-    # app_id = request.args.get('appid')
-    message = "Application Updated"
-    return redirect(url_for("gbac_manageapps_bp.gbac_apps", _external=True, _scheme=session[SESSION_INSTANCE_SETTINGS_KEY]["app_scheme"], message=message))
+    oidcclientid = request.args.get('oidcclientid')
+    oidcloginredirecturi = request.args.get('oidcloginredirecturi')
+    oidcapplabel = request.args.get('oidcapplabel')
+
```

Comment:
Not checking if `oidc_clientid`, `oidc_login_redirect_uri`, `oidc_app_label` are None. What happens when the variables are None or ""?

Reply:
Fixed.